### ISSUE TITLE
Summary of Meeting: Feb 26, 2020

### ISSUE THREAD

**Issue by braydentang1** [UBC-MDS/encodeR#7]
Hey guys, just a summary of everything we discussed so that Bronwyn knows what's up:

- We have recreated the repos as required. Everything should be there, I followed the steps in "The Whole Game" book. I think they are both correct but I may have mistakenly missed a step for the R package in particular so I'd appreciate if you guys look over it.
- We need to write our doc strings for both the R and Python functions. For some of the functions, we may incorporate the response variable and so we were thinking that each function should probably accept at least two arguments by default (train, test). The idea is to learn the encodings on the training set, and replace the original categorical columns in the training set with their encodings (or maybe specify a boolean argument to the function where the user can specify if they wish to drop the original columns or not). For the test set, we would join the learned encodings from the training set to the test set.
- Ideally we would probably follow the sklearn fit and fit_transform methods, etc. but this requires making Python classes and in R there is no equivalent. If you guys have any suggestions or hate this idea in general, feel free to say something. 
- For the doc strings, I think we need to keep running certain commands using poetry and documents() in Python/R respectively to actually render these documents and update the NAMESPACE files, etc.. The two books basically describe this but just a heads up.
I'll write the READMEs.
- The contributors.md in the Python repo file needs to be updated with your email addresses and names. The R setup doesn't automatically generate contributors.md, contributing.md or conduct.md (I think, providing I followed the instructions correctly) so we need to add those to the R repo (just copy and paste from the Python one probably)
- In the R repo, there is a DESCRIPTIONS file. You guys need to add your names and emails there as well I think.
